tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Builders/OrderSummaryDataBuilder.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using NHSD.BuyingCatalogue.Ordering.Domain;
+using NHSD.BuyingCatalouge.Ordering.Api.Testing.Data.Entities;
+using NHSD.BuyingCatalouge.Ordering.Api.Testing.Data.EntityBuilder;
+using CatalogueItemType = NHSD.BuyingCatalouge.Ordering.Api.Testing.Data.Data.CatalogueItemType;
```

Comment:
I think in one of the coding standards meeting we decided against using this type of syntax for a using statement. Is it possible to import like the statements above (without the alias)?

Reply:
I did it to pick up a enum from domain, i see know that it should have come from Testing.Data.Data .  removed